### PR TITLE
fix env variables for postgres. POSTGRES_PASSWORD required for image to start

### PR DESCRIPTION
**Pull request by davecramer**
as per docs here https://github.com/docker-library/docs/tree/master/postgres

### CODE REVIEW

File: kstreams-live-update/docker-compose-postgres.yaml [hunk 1]
```diff
@@ -30,9 +30,9 @@ services:
     environment:
      - PG_MODE=primary
      - PG_PRIMARY_PORT=5432
-     - PG_PRIMARY_USER=postgres
```
Comment (gunnarmorling) [inline kstreams-live-update/docker-compose-postgres.yaml]: Will PORT above need updating, too?

Comment (davecramer) [reply kstreams-live-update/docker-compose-postgres.yaml]: Seems to work fine, but probably because the port is the default port.

Comment (davecramer) [reply kstreams-live-update/docker-compose-postgres.yaml]: I don't see any reference to the port in the docs

**Comment by Naros** [comment]
This seems to be out-of-date, closing.  Thanks @davecramer!.

**Pull request closed without merging**